src/app/admin/admin/admin.component.html

```diff
@@ -0,0 +1,18 @@
+<div fxLayout="column" fxLayoutAlign="center center" style="padding-top: 35px;" style="border-color: violet; border-style: solid;">
+    <div fxLayout="row" style="border-color:red; border-style: solid; width: 800px;" >
+        <div fxFlex=50 fxLayoutAlign="start center" style="border-color:black; border-style: solid;">
```

Comment:
cannot this styling be moved to separate .css file and be mentioned  here with a class name?

Reply:
Similar to previous comment